### PR TITLE
[MIG][15.0] fieldservice_account

### PR DESCRIPTION
**Pull request by Vandan-OSI**

### CODE REVIEW

File: fieldservice_account/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    )
+
+    @api.depends("line_ids")
+    def _compute_fsm_order_ids(self):
+        for order in self:
+            orders = self.env["fsm.order"].search(
+                [("invoice_lines", "in", order.line_ids.ids)]
+            )
+            order.fsm_order_ids = orders
+            order.fsm_order_count = len(order.fsm_order_ids)
```
Comment (Murtaza-OSI) [inline fieldservice_account/models/account_move.py:27]: Single write to this two line

File: fieldservice_account/models/fsm_order.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class FSMOrder(models.Model):
+    _inherit = "fsm.order"
+
+    invoice_lines = fields.Many2many(
```
Comment (Murtaza-OSI) [inline fieldservice_account/models/fsm_order.py:9]: Rename to invoice_lines_ids

File: fieldservice_account/models/fsm_order.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    )
+
+    @api.depends("invoice_lines")
+    def _compute_get_invoiced(self):
+        for order in self:
+            invoices = order.invoice_lines.mapped("move_id").filtered(
+                lambda r: r.move_type in ("out_invoice", "out_refund")
+            )
+            order.invoice_ids = invoices
+            order.invoice_count = len(invoices)
```
Comment (Murtaza-OSI) [inline fieldservice_account/models/fsm_order.py:40]: Single write to this two line

File: fieldservice_account/README.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Ffield--service-lightgray.png?logo=github
+    :target: https://github.com/OCA/field-service/tree/14.0/fieldservice_account
```
Comment (Murtaza-OSI) [inline fieldservice_account/README.rst:17]: Generate Readme using maintainer-tools 
https://github.com/OCA/maintainer-tools

**Comment by max3903** [comment]
@murtuzasaleh @Freni-OSI Can you get this one fixed so we can merge it?

**Comment by Freni-OSI** [comment]
> @murtuzasaleh @Freni-OSI Can you get this one fixed so we can merge it?

Yes @max3903 , I'm working on it.

**Review by murtuzasaleh — APPROVED** [review]

**Comment by Freni-OSI** [comment]
Can you proceed to merge this PR? 
CC: @max3903 @dreispt @brian10048

**Pull request merged**